framework/src/main/java/org/checkerframework/framework/stub/AnnotationFileParser.java

```diff
@@ -623,34 +705,46 @@ private void processPackage(PackageDeclaration packDecl) {
      * @param typeDecl the type declaration to process
      * @param outertypeName the name of the containing class, when processing a nested class;
      *     otherwise null
+     * @param classTree the tree corresponding to typeDecl if processing an ajava file, null
+     *     otherwise
+     * @return a list of types variables for {@code typeDecl}. These should be removed after
```

Comment:
Removed from where?

Also, many clients ignore the return value.  Is that OK?  Is ignoring the return value OK exactly when the last argument is null?  Or maybe when isParsingStubFile is false:?

Should a similar note be added to `processCallableDeclaration`?